src/sql/query/insert_employee_data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO employee (client_employee_id, first_name, last_name, department_id, manager_employee_id, salary, hire_date, terminated_date, terminated_reason, dob, fte, weekly_hours, role) 
+VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s);
+
+    
```

Comment:
You are inserting record row by row, why is it so? Is there another way to do it? You already have raw data in your database and only need to transform them and store in the destination table. Is there way to move data from one table to another in SQL query itself?

Reply:
Hajur, didi! If eutai db ma xa bhane insert mai select garera halna milxa. In this case both raw and employee are in same db, so we can do this as well.